docs/penalty-box.raml

```diff
@@ -20,9 +20,9 @@ documentation:
   post:
     description: |
       Post info on a request to see if request is allowed. If allowed Penalty Box appends keep track of it to enforce rate limits.
-      The response body contains three values: X-Rate-Limit-Limit, X-Rate-Limit-Remaining, X-Rate-Limit-Reset. The first value contains the rate limit for
-      this apps' key. The second value contains the number of requests left in this minute. The third value contains time (epoch ms) till the rate limit is
-      reset
+      The response body contains three values: limit, remaining, success, and reset. The first value contains the rate limit for
+      this apps' key. The second value contains the number of requests left in this minute. The third value contains whether or not
+      the request was accepted. The final value contains time (epoch ms) till the rate limit is reset.
```

Comment:
Also the name of the `success` key and its wording in the description string are not super clear. Saying the request was not accepted makes it sound like the server is 400ing, but really you are referring to the rate limit threshold being crossed. Maybe `is_rate_limited` or something more targeted.